build.sbt

```diff
@@ -31,30 +27,36 @@ developers := List(
   Developer(id = "annebyrne", name = "Anne Byrne", email = "", url = url("https://github.com/annebyrne"))
 )
 
-releaseProcess := Seq(
+publishTo := sonatypePublishToBundle.value
+
+import ReleaseTransformations._
+
+releaseCrossBuild := true // true if you cross-build the project for multiple Scala versions
+releaseProcess := Seq[ReleaseStep](
   checkSnapshotDependencies,
   inquireVersions,
   runClean,
   runTest,
   setReleaseVersion,
   commitReleaseVersion,
   tagRelease,
-  publishArtifacts,
+  // For non cross-build projects, use releaseStepCommand("publishSigned")
+  releaseStepCommandAndRemaining("+publishSigned"),
+  releaseStepCommand("sonatypeBundleRelease"),
   setNextVersion,
   commitNextVersion,
-  releaseStepCommand("sonatypeRelease"),
   pushChanges
 )
 resolvers += Resolver.sonatypeRepo("releases")
 
 libraryDependencies ++= Seq(
   "io.circe" %% "circe-core" % circeVersion,
   "io.circe" %% "circe-generic" % circeVersion,
-  "org.apache.thrift" % "libthrift" % "0.12.0",
-  "com.twitter" %% "scrooge-core" % "19.9.0",
+  "org.apache.thrift" % "libthrift" % "0.13.0",
```

Comment:
ooooh I wasn't aware there was a new version. [The changes](https://github.com/apache/thrift/blob/0.13.0/CHANGES.md) are quite substantial for the Java language, so I recommend testing the library on another project before merging.

Reply:
Thanks - I've done that against a local build of Ophan now, we have [`AcquisitionExtractorTest`](https://github.com/guardian/ophan/blob/3ad1a0255d/the-slab/test/extractors/AcquisitionExtractorTest.scala#L29) which tests the Fezziwig-generated Circe-decoder used in [`AcquisitionExtractor`](https://github.com/guardian/ophan/blob/3ad1a0255d38/the-slab/app/extractors/AcquisitionExtractor.scala#L3-L11), and all tests passed :+1: